rboot-sampleproject/main.c

```diff
@@ -18,9 +18,6 @@
 
 static os_timer_t network_timer;
 
-void ICACHE_FLASH_ATTR user_rf_pre_init() {

```

Comment:
I have a standalone compiler (with the SDK included) while you apperantly have a compiler with the SDK excluded. Maybe that is causing for these differences between your and my setup. If that is true then maybe the SDK pointing should also move to that personal Makefile. And depending on the SDK being set or not supply the user_rf_pre_init. I guess we will know more once you tried to compile without the user_rf_pre_init, because it could also be the 1.2.0 version thing. Then we could check for that and put an #ifdef around user_rf_pre_init depening on the SDK version.